src/components/anchor.tsx

```diff
@@ -17,12 +17,13 @@ interface Props {
 	children?: ReactNode;
 	format: Format;
 	className?: SerializedStyles;
+	useDarkMode?: boolean;
 }
 
-const styles = css`
+const styles = (useDarkMode: boolean): SerializedStyles => css`
```

Comment:
Nice one @abeddow91 , can I suggest we name it something else, `useDarkMode` sounds like a React hook. Maybe `supportsDarkMode` or something along those lines?

Reply:
Makes sense, will do :)